code/game/objects/items/devices/flashlight.dm

```diff
@@ -207,6 +207,7 @@
 	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
 	force = 9 // Not as good as a stun baton.
 	brightness_on = 5 // A little better than the standard flashlight.
+	total_mass = 0.4
```

Comment:
No, these are already small items and don't consume much stamina to swing.

Reply:
This one is ironicly realistic. As stated by Eswords commits. This one is staying

Reply:
Believe me, the original concept of total mass in pounds also used for stamina consumption per attack is pretty hard to maintain and balance. No excuse.

Reply:
And again, this one is a case of Bhijn's commits. Its relistic and higher RP.

Reply:
You have no idea of what you are talking about. Even in a realistic setting, your impromptu baton flashlight would at least be one pound and half, but again you seem to have never held a decently long heavy duty flashlight in your whole life to know that they must have some weight to pack a punch.